movo_demos/scripts/cartesian_control.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+import sys
+import copy
+import rospy
+import moveit_commander
+import moveit_msgs.msg
+import geometry_msgs.msg
+from math import pi
+from std_msgs.msg import String
+from sensor_msgs.msg import JointState
+from moveit_commander.conversions import pose_to_list
+from trajectory_msgs.msg import (JointTrajectoryPoint,)
+
+"""
+This class takes root in the moveit_ Move Group Python Interface. This code is useful because someone can plan trajectories and movements with only a python code. The link [https://ros-planning.github.io/moveit_tutorials/doc/move_group_python_interface/move_group_python_interface_tutorial.html].
+
+It has been adapted for MOVO, or at least, it should work on MOVO. 
```

Comment:
But does it? What does this comment mean?

Reply:
It is to give credit to code found on MoveIt. I have taken the move_group_python interface and modified it specifically for Movo